src/pages/documentation/media-for-edge-network/index.md

```diff
@@ -1,17 +1,24 @@
 
-<InlineAlert variant="info" slots="text"/>
+## Before starting
 
-This extension requires the setting up of Datastreams with Media forwarding enabled, Dataset, XDM Schema and other AEP settings. Please follow this `backendDocumentationLink` before configuring and implementing the SDK.  
+### Configure and Setup Media with Experience Platform
 
+1. [Define a report suite](https://experienceleague.adobe.com/docs/media-analytics/using/implementation/media-sdk/setup/implementation-edge.html?lang=en#define-a-report-suite)
+2. [Set up the schema in Adobe Experience Platform](https://experienceleague.adobe.com/docs/media-analytics/using/implementation/media-sdk/setup/implementation-edge.html?lang=en#set-up-the-schema-in-adobe-experience-platform)
+3. [Create a dataset in Adobe Experience Platform](https://experienceleague.adobe.com/docs/media-analytics/using/implementation/media-sdk/setup/implementation-edge.html?lang=en#create-a-dataset-in-adobe-experience-platform)
+4. [Configure a datastream in Adobe Experience Platform](https://experienceleague.adobe.com/docs/media-analytics/using/implementation/media-sdk/setup/implementation-edge.html?lang=en#configure-a-datastream-in-adobe-experience-platform)
 
-## Configure Edge Media Analytics extension in the Data Collection UI
+Follow the full guide for setting up [Media Analytics for Edge Network with Experience Platform Edge](https://experienceleague.adobe.com/docs/media-analytics/using/implementation/media-sdk/setup/implementation-edge.html?lang=en) before configuring and implementing the SDK.
 
-<InlineAlert variant="info" slots="text"/>
+### Configure and Install Dependencies
 
-Media for Edge Network depends on Edge and Edge Identity extensions. Make sure to [configure the Edge extension in Data Collection UI](https://developer.adobe.com/client-sdks/documentation/edge-network/#configure-the-edge-network-extension-in-data-collection-ui) and [configure the Edge Identity extension in Data Collection UI](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/#configure-the-identity-extension-in-the-data-collection-ui) before proceeding.
+Media for Edge Network requires Edge and Edge Identity extensions. Make sure to [configure the Edge extension in Data Collection UI](https://developer.adobe.com/client-sdks/documentation/edge-network/#configure-the-edge-network-extension-in-data-collection-ui) and [configure the Edge Identity extension in Data Collection UI](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/#configure-the-identity-extension-in-the-data-collection-ui) before proceeding.
+
+## Configure Edge Media Analytics extension in the Data Collection UI
 
 1. In the Data Collection UI, select the **Extensions** tab.
 2. On the **Catalog** tab, locate the **Adobe Edge Media Analytics for Audio and Video** extension, and select **Install**.
```

Comment:
Type the extension settings. For more information, see [Configure Media Analytics Extension](http://localhost:8000/documentation/media-for-edge-network/#configure-media-analytics-extension).

Suppose to be Configure the Edge Media Analytics extension and link?